mxcube3/routes/Login.py

```diff
@@ -125,7 +125,6 @@ def loginInfo():
 
     login_info = login_info["loginRes"] if login_info is not None else {}
     login_info = limsutils.convert_to_dict(login_info)
-    limsutils.update_mxcube_session(login_id, login_info)
```

Comment:
see https://github.com/mxcube/mxcube3/blob/master/mxcube3/routes/limsutils.py#L80

Of course... there could be a bug